### PR TITLE
Add case list menu item back into app manager v2

### PR DESCRIPTION
**Pull request by orangejenny**
Would be great to integrate this better into app manager, but doing so isn't a priority for getting users onboarded, so for now it's just going to be one of the "advanced" app manager features.

@biyeun / @esoergel

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_settings.html [hunk 1]
```diff
@@ -126,6 +126,8 @@ <h4 class="panel-title panel-title-nolink">{% trans "Advanced Settings" %}</h4>
             </div>
             <div class="panel-body">
               {% include 'app_manager/v2/partials/case_list_form_setting.html' %}
+
+              {% include 'app_manager/v1/partials/case_list_setting.html' with LABEL="Case List Menu Item" DESC="An item in the module's menu that lets you browse the case list without moving on to fill out a form." SLUG="case_list" case_list=module.case_list %}
```
Comment (esoergel) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_settings.html]: should this be `v1` or `v2`?

Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_settings.html]: yeah this should be a v2 template

Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_settings.html]: seems more verbose right now, but will make people more aware of making changes and make things easier when we move over to v2

Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_settings.html]: makes sense, updated

**Review by biyeun — APPROVED** [review]

**Pull request merged**